driver/network_driver.go

```diff
@@ -473,9 +473,46 @@ func (d NetworkDriver) Join(request *network.JoinRequest) (*network.JoinResponse
 		NextHop:     "",
 	})
 
+	// NOTE(jkoelker) Check if we should attempt ipv6 config
+	ipv6Enabled := true
+
+	os.Setenv("DOCKER_API_VERSION", "1.25")
+	dockerCli, err := dockerClient.NewEnvClient()
```

Comment:
What's the reasoning for pinning the API version?

Reply:
No real reason, it was pinned below (and using the deprecated `NewEnvClient` init function), and I just copied that. Eventually when we can overhaul this, we should look into refactoring the docker methods out into a wrapper type, so we can manage the context better and retry a bit better (below its using multiple `goto`s ;( ).